### PR TITLE
Api cli close modal on linode create

### PR DESCRIPTION
**Pull request by cpathipa**
## Description 📝
Address UX Feedback
Automatically closes commands modal upon creating Linode from terminal.

## How to test 🧪

- Go to Linode create page and fill required fields.
- Click on Create Command Line button.
- Copy command and execute in terminal.
- Observe behavior of closing modal and navigating user back to Linode's page.

Test creating Linode, clone Linode, Deploy Linode from backup

**How do I run relevant unit or e2e tests?**

- yarn test
- yarn cy:run

### CODE REVIEW

**Review by bnussman-akamai — APPROVED** [review]
Redirect works as expected! Nice changes to the copy icon styles too 🚀

File: packages/manager/src/features/linodes/LinodesCreate/ApiAwarenessModal/index.tsx [hunk 1]
```diff
@@ -95,6 +111,13 @@ const ApiAwarenessModal = (props: Props) => {
     });
   };
 
+  useEffect(() => {
+    if (isLinodeCreated && isOpen) {
+      onClose();
+      history.replace('/linodes');
```
Comment (bnussman-akamai) [inline packages/manager/src/features/linodes/LinodesCreate/ApiAwarenessModal/index.tsx:117]: I wonder if we should send the user to the Linodes Details page (`/linodes/:id`). If a user with hundreds of Linodes got sent back to the Linodes landing page, they may not see the new Linode if it is not on the current page.

**Review by jdamore-linode — APPROVED** [review]
Echoing @bnussman-akamai's suggestion, but otherwise this looks really good! Confirmed the redirect occurs after creating the Linode (and that no redirect occurs if the Linode created via CLI's label doesn't match the expected label). In the middle of running e2e tests, but I'm assuming they'll pass and will circle back if I see any suspicious failures.

File: packages/manager/src/features/linodes/LinodesCreate/ApiAwarenessModal/index.tsx (hunk 1 above)
Comment (cpathipa) [reply packages/manager/src/features/linodes/LinodesCreate/ApiAwarenessModal/index.tsx]: Make sense, updated as suggested.

**Comment by cpathipa** [comment]
Thank you @bnussman-akamai and @jdamore-linode.

**Pull request merged**